src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -84,28 +84,40 @@ public function __construct(ContainerBuilder $container, PhpFileLoader $loader,
 
 /**
  * Creates a service reference.
+ *
+ * @deprecated since Symfony 5.1, use service() instead.
  */
 function ref(string $id): ReferenceConfigurator
+{
+    trigger_deprecation('symfony/dependency-injection', '5.1', '"%s()" is deprecated, use "service()" instead.', __FUNCTION__);
+
+    return new ReferenceConfigurator($id);
+}
+
+/**
+ * Creates a service reference.
+ */
+function service(string $id): ReferenceConfigurator
```

Comment:
```suggestion
function service(string $serviceId): ReferenceConfigurator
```

We're big on the php format to help with auto-complete. This will help them understanding the meaning also.